fastats/utilities/pre_processing.py

```diff
@@ -0,0 +1,50 @@
+from numpy import empty_like, mean, std, sqrt
+from numpy import min as np_min
+from numpy import max as np_max
+
+
+def standard_scale(data, ddof=0):
```

Comment:
Nice addition with the scaling functions - my feeling is that scaling functions fit perfectly within the fastats substitution concept, so people may want to swap them out with `my_func(..., scale=standard)` or `my_func(..., scale=min_max)` - what do you think?